apps/ledger-live-mobile/src/analytics/segment.ts

```diff
@@ -337,14 +337,53 @@ export const useAnalytics = () => {
     page,
   };
 };
+
+/**
+ * Track an event which will have the name `Page ${category}${name ? " " + name : ""}`.
+ * Extra logic to update the route names used in "screen" and "source"
+ * properties of further events can be optionally enabled with the parameters
+ * `updateRoutes` and `refreshSource`.
+ */
 export const screen = async (
```

Comment:
If the parameters of this `screen` function are all optional, it can be interesting to put them in an object, so the consumer can choose which parameter to set and not set:

```
screen({category, name, properties, etc. }: { category?: string; name?: string | null; properties?: etc. }
```

WDYT ?

Reply:
Usages of this are pretty rare/exceptional so I think it's not too inconvenient to keep the signature like this for now.